.gitignore

```diff
@@ -36,8 +36,13 @@ yarn-debug.log*
 
 .env*
 
-# Ignore docker related files
-/vendor
+# Ignore docker related files, but don't ignore active admin's css specifications
+vendor/**
+!vendor/assets/
+!vendor/assets/stylesheets/
+!vendor/assets/stylesheets/**
+!vendor/assets/stylesheets/active_admin.scss
```

Comment:
1. This is redundant. `stylesheets/**` matches everything in the folder, you don't need to explicitly mention `active_admin.scss` as well.

   Based on the comment, I think we probably would only need `!vendor/assets/stylesheets/active_admin.scss` to explicitly include that one file.

2. I don't think `.gitignore` is the right place to handle this edge case in the first place. `git add` has a [`--force` flag](vendor/assets/stylesheets/active_admin.scss) to explicitly add an ignored file. Once a file has been committed, git will continue to track it even if `.gitignore` says otherwise.